src/Instance/InstanceEdit/InstanceEdit.js

```diff
@@ -69,11 +75,19 @@ const InstanceEdit = ({
     setHttpError(parsedError);
   };
 
-  const onSubmit = useCallback((updatedInstance) => {
-    return mutator.instanceEdit.PUT(marshalInstance(updatedInstance, identifierTypesByName))
-      .then(() => onSuccess(updatedInstance))
-      .catch(onError);
-  }, [identifierTypesByName, onError]);
+  const isCentralTenant = checkIfUserInCentralTenant(stripes);
+  const tenantId = isCentralTenant || instance?.shared ? stripes.user.user.consortium.centralTenantId : stripes.okapi.tenant;
```

Comment:
Did you mean?
```suggestion
  const tenantId = (isCentralTenant || instance?.shared) ? stripes.user.user.consortium.centralTenantId : stripes.okapi.tenant;
```

Because currently it looks like it returns `true` when `isCentralTenant` is true

Reply:
I guess it's better to replace it with:
```javascript
const tenantId = (isMemberTenant && instance?.shared) ? stripes.user.user.consortium.centralTenantId : stripes.okapi.tenant;
``` 

Because you need to change the tenant ID header only for member tenant when it edits a shared instance, in case when a central tenant edits the instance it will send stripes.okapi.tenant (that equals central tenant id)